src/Admin/CommentAdmin.php

```diff
@@ -3,22 +3,19 @@
 namespace SilverStripe\Comments\Admin;
 
 use SilverStripe\Admin\LeftAndMain;
-use SilverStripe\Comments\Admin\CommentsGridField;
 use SilverStripe\Comments\Model\Comment;
-use SilverStripe\Forms\Tab;
-use SilverStripe\Forms\TabSet;
 use SilverStripe\Forms\FieldList;
 use SilverStripe\Forms\Form;
-use SilverStripe\Security\PermissionProvider;
+use SilverStripe\Forms\Tab;
+use SilverStripe\Forms\TabSet;
 use SilverStripe\Security\Security;
-use SilverStripe\View\SSViewer;
 
 /**
  * Comment administration system within the CMS
  *
  * @package comments
  */
-class CommentAdmin extends LeftAndMain implements PermissionProvider
```

Comment:
How come this was removed?

Reply:
The parent class implements it already. The change is unclear, sorry - the code is redundant